src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -47,6 +60,20 @@ export class RundownComponent implements OnInit, OnDestroy {
     this.startOffsetsInMsFromPlayheadForSegments = this.getStartOffsetsInMsFromPlayheadForSegments(rundownTimingContext)
   }
 
+  private onActionsChanged(actions: Action[]): void {
+    this.videoClipActions = actions.filter((action): action is Tv2VideoClipAction => {
+      return action && (<Tv2Action>action).metadata?.contentType === Tv2ActionContentType.VIDEO_CLIP
+    })
```

Comment:
When you have a single expression to return from a lambda function, the curly brackets and the return keyword are redundant. The check for whether the action is a truthy value should not be necessary. And the type assertion should be changed to `action as Tv2Action`. So in all this can be simplified to:
```typescript
    this.videoClipActions = actions.filter((action): action is Tv2VideoClipAction => (action as Tv2Action).metadata?.contentType === Tv2ActionContentType.VIDEO_CLIP)
```